prediction_market_agent/agents/omen_cleaner_agent/deploy.py

```diff
@@ -48,4 +65,42 @@ def resolve_finalized_markets(self, api_keys: APIKeys) -> list[HexAddress]:
             )
             if market.id not in IGNORED_FOR_RESOLUTION
         ]
+        logger.info(
+            f"Found {len(finalized_unresolved_markets)} finalized unresolved markets."
+        )
         return resolve_markets(api_keys, finalized_unresolved_markets)
+
+    @observe()
+    def generate_missing_images(
+        self, api_keys: APIKeys
+    ) -> dict[HexAddress, IPFSCIDVersion0 | None]:
+        logger.info("Generating missing images.")
+        recently_created_markets = OmenSubgraphHandler().get_omen_binary_markets(
+            limit=None,
+            # Get only serious markets with a reasonable liquidity.
+            liquidity_bigger_than=xdai_to_wei(xdai_type(5)),
+            # Get only markets created roughly since the last run.
+            created_after=utcnow() - timedelta(days=2),
+        )
+        logger.info(
+            f"Found {len(recently_created_markets)} recent markets without an image."
+        )
+        generated_image_mapping: dict[HexAddress, IPFSCIDVersion0 | None] = {}
```

Comment:
Any reason to generate a mapping if you don't plan on using it anywhere?

Reply:
It's observed in Langfuse

Reply:
ooo interesting